srcpkgs/cgo/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'cgo'
+pkgname=cgo
+version=0.5.0
+revision=1
+build_style=gnu-makefile
+short_desc="Terminal based gopher client"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
+license="ISC"
+homepage="https://github.com/kieselsteini/cgo"
+distfiles="https://github.com/kieselsteini/${pkgname}/archive/v${version}.tar.gz"
```

Comment:
Please write `cgo` instead of `${pkgname}`.

Reply:
done.